src/app/features/courses/components/course-card/course-card.component.ts

```diff
@@ -0,0 +1,38 @@
+import { Component, Input, OnInit } from '@angular/core';
+
+export type CourseCard = {
+  id: string,
+  title: string,
+  description: string,
+  creationDate: string,
+  duration: number,
+  authors: Array<string>
+}
+
+@Component({
+  selector: 'app-course-card',
+  templateUrl: './course-card.component.html',
+  styleUrls: ['./course-card.component.scss']
+})
+export class CourseCardComponent implements OnInit {
+  @Input() card?: CourseCard;
+
+  formattedDuration: string;
```

Comment:
you could assign the value here and omit type annotation (it will be applied by the value)

`formattedDuration = '';`